src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -300,23 +339,18 @@ public void visitToken(DetailAST ast) {
      * @param ast input DetailAST node.
      * @return true if it is an ignore situation found for given input DetailAST
      *         node.
-     * @noinspection SimplifiableIfStatement
      */
     private boolean isIgnoreSituation(DetailAST ast) {
         final DetailAST modifiers = ast.getFirstChild();
 
         final boolean result;
         if (ast.getType() == TokenTypes.VARIABLE_DEF) {
-            if ((ignoreFinal || ignoreStatic)
```

Comment:
not sure why this logic is removed. In `hasIgnoredModifiers` method if ignoreStaticFinal is true, we still ignore items if either static or final present, but for interface declaration it removed. Why?

Reply:
The intention here is that if `ignoreStaticFinal` has been explicitly set to `false`, we should respect that over `ignoreStatic` or `ignoreFinal`. So for interfaces where everything is implicitly `static final`, the only thing we need to check is `ignoreStaticFinal`.

> In `hasIgnoredModifiers` method if ignoreStaticFinal is true, we still ignore items if either static or final present

If both modifiers are present and `ignoreStaticFinal` is also true, it will return true. If only one modifier is present, it is checked against the respective ignore property instead.

`hasIgnoredModifiers` needs to handle the more general case where only one modifier is present, but inside interfaces there is no need for that logic.